pkg/volume/plugins.go

```diff
@@ -42,6 +42,13 @@ type VolumeOptions struct {
 	RootContext string
 }
 
+// System is an interface that abstracts certain operations on a host operating
+// system that a volume might want to execute.
+type System interface {
```

Comment:
Done. Added ```GetMounter() and ```GetWriter``` to ```VolumeHost```. I'm using ```GetWriter()``` instead of ```WriteFile()``` so that it will be easier to extend it to other operations (like symbolic links which are used in downward API plugin).